php-checks/src/test/resources/checks/phpunit/NoTestInTestClassCheck.php

```diff
@@ -26,3 +26,26 @@ private function testFoo() {}
 }
 
 abstract class MyTest extends TestCase {} // OK
+
+class MyAttribute0 extends TestCase { // Noncompliant {{Add some tests to this class.}}
+//    ^^^^^^^^^^^^
+  #[NotTestAttribute]
+  public function foo() {}
+}
+
+class MyAttribute1 extends TestCase {
+  #[Test]
+  public function foo() {}
+}
+
+class MyAttribute2 extends TestCase { // Noncompliant {{Add some tests to this class.}}
+//    ^^^^^^^^^^^^
+  #[TestDox('It does something')]
+  public function foo() {}
+}
+
+class MyAttribute3 extends TestCase {
```

Comment:
I would add additional test cases with the Attributes:

-  `[#\Foo\Test]`
-  `[#\Test]` 
- `[#Foo\Test]`
- `[#Test('foobar')]`.

Please check if these cases also behave as expected.

Reply:
Should me more like `#[\Foo\Test]`